contracts/Comptroller.sol

```diff
@@ -1103,7 +1103,7 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerE
      */
     function updateCompSupplyIndex(address cToken) internal {
         CompMarketState storage supplyState = compSupplyState[cToken];
-        uint supplySpeed = compSpeeds[cToken];
+        uint supplySpeed = compBorrowSpeeds[cToken];
```

Comment:
Sorry if it's just my misunderstanding, but setting `supplySpeed` to `compBorrowSpeeds[cToken]` and `borrowSpeed` to `compSupplySpeeds[cToken]` looks backwards at first glance. Assuming it's not a mistake, might be helpful to briefly explain these assignments with a comment?

Reply:
Nice eye. This problem was fixed in f9d129d607047c807d30b7c352cfda8f74650d2b.